src/api_routes_http.rs

```diff
@@ -297,8 +297,8 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
       .service(
         web::scope("/user")
           .wrap(rate_limit.import_user_settings())
-          .route("/export", web::get().to(export_user_backup))
-          .route("/import", web::get().to(import_user_backup)),
+          .route("/export", web::get().to(export_settings))
```

Comment:
The route urls should maybe change too, in case we add a `user/export` which includes everything.

Reply:
Like how?

Reply:
/export_settings and import_settings

Reply:
done